src/gui/GuiRunner.cc

```diff
@@ -77,15 +86,8 @@ void GuiRunner::OnPluginAdded(const QString &_objectName)
 }
 
 /////////////////////////////////////////////////
-void GuiRunner::OnStateService(const msgs::SerializedStepMap &_res,
-    const bool _result)
+void GuiRunner::OnStateAsyncService(const msgs::SerializedStepMap &_res)
 {
-  if (!_result)
-  {
-    ignerr << "Service call failed for [" << this->stateTopic << "]"
-           << std::endl;
-    return;
-  }
   this->OnState(_res);
```

Comment:
At this point, maybe we want to unadvertise the `StateAsyncService`? Just to keep the list of services cleaner and maybe alleviate a bit the transport.

Reply:
yep, unadvertised service. 14472ba